packages/alert-dialog/README.md

```diff
@@ -0,0 +1,308 @@
+    >
+        Cancel
+    </sp-button>
+    <sp-button
+        slot="button"
+        id="confirmButton"
+        variant="primary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Continue
+    </sp-button>
+</sp-alert-dialog>
+```
+
+### Error
+
+Error alert dialogs communicate critical information about an issue that a user needs to acknowledge.
+
+```html
+<sp-alert-dialog variant="error">
+    <h2 slot="heading">Unable to share</h2>
+    An error occured while sharing your project. Please verify the email address
+    and try again.
+    <sp-button
+        slot="button"
+        id="confirmButton"
+        variant="primary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Continue
+    </sp-button>
+</sp-alert-dialog>
+```
+
+### Destructive
+
+Destructive alert dialogs are for when a user needs to confirm an action that will impact their data or experience in a potentially negative way, such as deleting files or contacts.
+
+```html
+<sp-alert-dialog variant="destructive">
+    <h2 slot="heading">Delete 3 documents?</h2>
+    Are you sure you want to delete the 3 selected documents?
+    <sp-button
+        slot="button"
+        id="cancelButton"
+        variant="secondary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Cancel
+    </sp-button>
+    <sp-button
+        slot="button"
+        id="confirmButton"
+        variant="negative"
+        treatment="fill"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Delete
+    </sp-button>
+</sp-alert-dialog>
+```
+
+### Secondary Button
+
+An alert dialog can have a total of 3 buttons if the secondary outline button label is defined.
+
+```html
+<sp-alert-dialog variant="secondary">
+    <h2 slot="heading">Rate this app</h2>
+    If you enjoy our app, would you mind taking a moment to rate it?
+    <sp-button
+        slot="button"
+        id="secondaryButton"
+        variant="secondary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        No, thanks
+    </sp-button>
+    <sp-button
+        slot="button"
+        id="cancelButton"
+        variant="secondary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Remind me later
+    </sp-button>
+    <sp-button
+        slot="button"
+        id="confirmButton"
+        variant="primary"
+        treatment="outline"
+        onclick="this.dispatchEvent(new Event('close', { bubbles: true, composed: true }));"
+    >
+        Rate now
+    </sp-button>
+</sp-alert-dialog>
+```
+
+### Scroll
+
+```html
+<sp-alert-dialog variant="scroll">
```

Comment:
Is `scroll` a variant, or a thing that just happens?

Reply:
yes scroll is a variant: https://opensource.adobe.com/spectrum-css/alertdialog.html#scroll

Reply:
Help me understand what's different in that "variant". It appears to be an example of a feature, which points to why an override was necessary to deliver it.